content/en/serverless/step_functions/installation.md

```diff
@@ -185,13 +188,19 @@ Alternatively, if you have business logic defined in the payload, you could also
 
 If you have not yet instrumented your Lambda functions to send traces, you can [follow the steps to add the Lambda layer for your preferred runtime][3].
 
+[1]: https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-json-manipulate
+[2]: https://docs.aws.amazon.com/step-functions/latest/dg/input-output-contextobject.html
+[3]: /logs/guide/forwarder/?tab=cloudformation#installation
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Link Step Function traces to nested Step Function traces" level="h4" %}}
+TODO
+{{% /collapse-content %}} 
```

Comment:
```suggestion
<!--
{{% collapse-content title="Link Step Function traces to nested Step Function traces" level="h4" %}}
TODO
{{% /collapse-content %}} 
-->
```
Commenting this out until there's useful content to present to the user.

Reply:
Make sense! Fixed.